diff --git a/CMakeLists.txt b/CMakeLists.txt
index 15768928..b8e9d457 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -122,14 +122,6 @@ if(BUILD_PYTHON_INTERFACE)
   find_package(Boost REQUIRED COMPONENTS system)
 endif(BUILD_PYTHON_INTERFACE)
 
-if(Boost_VERSION_STRING VERSION_LESS 1.81)
-  # Default C++ version should be C++11
-  CHECK_MINIMAL_CXX_STANDARD(11 ENFORCE)
-else()
-  # Boost.Math will be C++14 starting in July 2023 (Boost 1.82 release)
-  CHECK_MINIMAL_CXX_STANDARD(14 ENFORCE)
-endif()
-
 # Optional dependencies
 ADD_PROJECT_DEPENDENCY(octomap PKG_CONFIG_REQUIRES "octomap >= 1.6")
 if(octomap_FOUND)
diff --git a/cmake-modules/Findassimp.cmake b/cmake-modules/Findassimp.cmake
index 0b3dba21..ec7fb94e 100644
--- a/cmake-modules/Findassimp.cmake
+++ b/cmake-modules/Findassimp.cmake
@@ -14,7 +14,7 @@ if(WIN32)
 		HINTS
 			${ASSIMP_ROOT_DIR}/include
 	)
-  	SET(assimp_INCLUDE_DIRS ${ASSIMP_INCLUDE_DIR})
+  	SET(assimp_INCLUDE_DIR ${ASSIMP_INCLUDE_DIR})
 
 	if(MSVC12)
 		set(ASSIMP_MSVC_VERSIONS "vc120")
@@ -76,11 +76,11 @@ if(WIN32)
 else(WIN32)
 
 	find_path(
-	  assimp_INCLUDE_DIRS
+	  assimp_INCLUDE_DIR
 	  NAMES assimp/postprocess.h assimp/scene.h assimp/version.h assimp/config.h assimp/cimport.h
 	  PATHS /usr/local/include
 	  PATHS /usr/include/
-		HINTS	${ASSIMP_ROOT_DIR}/include
+	  HINTS	${ASSIMP_ROOT_DIR}/include
 	)
 
 	find_library(
@@ -89,15 +89,16 @@ else(WIN32)
 	  PATHS /usr/local/lib/
 	  PATHS /usr/lib64/
 	  PATHS /usr/lib/
+	  HINTS	${ASSIMP_ROOT_DIR}/lib
 	)
 
-	if (assimp_INCLUDE_DIRS AND assimp_LIBRARIES)
+	if (assimp_INCLUDE_DIR AND assimp_LIBRARIES)
 	  SET(assimp_FOUND TRUE)
-	ENDIF (assimp_INCLUDE_DIRS AND assimp_LIBRARIES)
+	ENDIF (assimp_INCLUDE_DIR AND assimp_LIBRARIES)
 
 	if (assimp_FOUND)
 	  if (NOT assimp_FIND_QUIETLY)
-		message(STATUS "Found asset importer library: ${assimp_LIBRARIES}")
+		message(STATUS "Found asset importer library")
 	  endif (NOT assimp_FIND_QUIETLY)
 	else (assimp_FOUND)
 	  if (assimp_FIND_REQUIRED)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 474e4a33..0fc771b3 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -227,6 +227,12 @@ IF(octomap_FOUND)
     TARGETS octomap
     LIBRARIES ${OCTOMAP_LIBRARIES}
     INCLUDE_DIRS ${OCTOMAP_INCLUDE_DIRS})
+  set_target_properties(
+    octomap PROPERTIES IMPORTED_LOCATION "${OCTOMAP_INCLUDE_DIRS}/../lib/liboctomap.so.1.9.8"
+  )
+  set_target_properties(
+    octomath PROPERTIES IMPORTED_LOCATION "${OCTOMAP_INCLUDE_DIRS}/../lib/liboctomath.so.1.9.8"
+  )
   target_compile_definitions (${LIBRARY_NAME} PUBLIC
     -DHPP_FCL_HAS_OCTOMAP
     -DHPP_FCL_HAVE_OCTOMAP
